inductiva/molecules/protein_solvation/README.md

```diff
@@ -38,25 +38,27 @@ To test this scenario we have available one example of a protein PDB file - [dow
 
 inductiva.api_key = "YOUR_API_KEY"
 
- # Initialize the scenario
- scenario = inductiva.molecules.ProteinSolvation(
-     protein_pdb = "alanine.pdb",
+# Initialize the scenario
+lysozyme = inductiva.molecules.examples.load_lysozyme()
+scenario = inductiva.molecules.ProteinSolvation(
+     protein_pdb = lysozyme,
      temperature = 300)
 
- # Run a simulation
- task = scenario.simulate(simulation_time_ns = 0.5)
+# Run a simulation
+task = scenario.simulate(simulation_time_ns = 0.05)
```

Comment:
If this still takes 5 mins I would decrease the time in another order of magnitude. Users should not have to wait so much

Reply:
I changed it for 0.01! I don't want to decrease it too much because the output will have very few frames.

Reply:
But maybe we should keep alanine and show more frames, decide what you think is best here... I know that your goal is for the post-processing part, but there you can explain and introduce the lysozyme for having the backbone plotted :)